src/elements/content-explorer/stories/tests/MetadataView-visual.stories.tsx

```diff
@@ -5,45 +5,63 @@ import ContentExplorer from '../../ContentExplorer';
 import { DEFAULT_HOSTNAME_API } from '../../../../constants';
 import { mockMetadata, mockSchema } from '../../../common/__mocks__/mockMetadata';
 
+// The intent behind relying on mockMetadata is to allow a developer to paste in their own metadata template schema for use with live API calls.
 const EID = '0';
-const templateName = 'templateName';
-const metadataSource = `enterprise_${EID}.${templateName}`;
-const metadataSourceFieldName = `metadata.${metadataSource}`;
+const templateScope = mockSchema.scope ?? `enterprise_${EID}`;
+const templateKey = mockSchema.templateKey ?? 'mdViewTemplate1';
```

Comment:
isnt the `mockSchema` always there why do we need to ?? ?

Reply:
I was tip-toeing the line between completely removing the old logic (developer passes in / manually updates hardcoded values) and using the new logic (rely solely on mockSchema). In my opinion this is less about a functional safety check and more about saying "we want you to use mockSchema, bit if you're editing locally, you can also just pass in hardcoded values".

Reply:
I'm happy to update it if you want, just tell me - I rely on you for what actually seems clear to another developer. I only have my own biased opinion on what is "clear"!